client/my-sites/domains/domain-management/edit-contact-info/form-card.jsx

```diff
@@ -283,8 +281,7 @@ class EditContactInfoFormCard extends Component {
 
 		const { email } = newContactDetails;
 		if ( updateWpcomEmail && email && this.props.currentUser.email !== email ) {
-			wpcom
-				.me()
+			wp.me()
```

Comment:
Interesting, the `wpcom.undocumented().me()` was inherited from `wpcom.me()`, and included all the methods... I was confused at first how the `.settings()` method could work at all.

Reply:
That part was pretty confusing, so ambiguous! You could use `wpcom.undocumented().me().settings()` or `wpcom.me().settings()` and they would both do the same thing! And having `.settings()` available for `wpcom.undocumented()` itself doesn't make it less confusing 😅